test/3rdparty/discover-0.4.0/README.txt

```diff
@@ -92,6 +92,11 @@ http://code.google.com/p/unittest-ext/source/browse/trunk/discover.py
 CHANGELOG
 =========
 
+2024/04/16 0.4.0
```

Comment:
Can you flag that this is a change was made by the WiredTiger team and doesn't come from the discover maintainers? Right now this looks like identical to the prior changelogs

Reply:
Sounds good, I've moved the WT changes to the start of the README file and added headers to make it clear. See changes here: [b869f07](https://github.com/wiredtiger/wiredtiger/pull/10491/commits/b869f07be7ac4b1a20f7864833f4abea449cb9be)
Please let me know if you think the current format is clear enough, happy to add additional notes if needed. :))

Reply:
Looks perfect thanks!